ui/apps/dashboard/src/app/actions.ts

```diff
@@ -0,0 +1,9 @@
+'use server';
```

Comment:
When the user collapses the nav we do that immediately client side state but also send it server side so that it persists if they refresh the page and so that any server rendered components are aware of it.